example/net.go

```diff
@@ -1,48 +1,48 @@
 package main
 
 import (
+	"context"
 	"fmt"
 	"github.com/project-receptor/receptor/pkg/backends"
 	"github.com/project-receptor/receptor/pkg/netceptor"
 	"io"
 	"net"
 	"os"
-	"sync"
 	"time"
 )
 
 /*
    This is an example of the use of Receptor as a Go library.
 */
 
-// Error handler that gets called for backend errors
-func handleError(err error, fatal bool) {
-	fmt.Printf("Error: %s\n", err)
-	if fatal {
-		os.Exit(1)
-	}
-}
-
 func main() {
+	// Create contexts so we can shut down the nodes
+	ctx1, _ := context.WithCancel(context.Background())
```

Comment:
I think this can be `context.Background()` since the netceptor adds the cancel function and we're not using this cancel function anyways. I think we could also pass a nil value and have the netceptor do everything.

Reply:
Yep, you're right